### ISSUE TITLE
<1024 ports without root

### ISSUE THREAD

**Issue by Hendrik-Huels** [coreos/fedora-coreos-tracker#1301]

**Comment by travier** [comment]
What's your use case? This is something easily changed in a Butane config / via Ignition.

We should probably ask for the change to be made in Fedora in general if we want to do it by default.

**Comment by dustymabe** [comment]
I agree. We wouldn't want to differ from the rest of Fedora on this issue. @Hendrik-Huels - please start a thread on the [fedora devel](https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/) mailing list to further the discussion. If it is received well you'd want to start a [change proposal](https://docs.fedoraproject.org/en-US/program_management/changes_policy/).
